control-plane/subcommand/server-acl-init/create_or_update.go

```diff
@@ -11,12 +11,63 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// addRoleAndBindingRule adds an ACLRole and ACLBindingRule which reference the authMethod.
```

Comment:
I moved the order of the method around so that the methods that are invoked from another are defined below the caller and hence the diff looks "big". The only actual change is adding support for global tokens.